src/pages/components/structured-list/usage.mdx

```diff
@@ -138,16 +162,86 @@ alignment.
+| _Enabled_  | When an action in the list is live but a user is not directly interacting with it. This is commonly referred to as the default or normal state of the component.                                                                  |
+| _Hover_    | When a user’s mouse cursor is hovering over the structured list item.                                                                                                                                                             |
+| _Focus_    | When a user tabs or clicks on a row, it becomes focused, indicating the user has successfully navigated to the component (except default structured list).                                                                        |
+| _Selected_ | When a user presses space or clicks on a structured list row to select it.                                                                                                                                                        |
+| _Disabled_ | When the user is not allowed to interact with a selectable structured list due to either permissions, dependencies, or pre-requisites.                                                                                            |
+| _Skeleton_ | When the user can review but not modify the component. This state removes all interactive functions like the disabled state but can still be focusable, accessible by screen readers, and passes visual contrast for readability. |
+
 ## Interactions
 
 #### Selectable structured list
 
 - Only one item can be selected from the list.
-- By default, one option should be selected.
+- By default, no option should be selected.
 - If you need to select multiple items, use a
   [data table.](/components/data-table/usage)
 - When the user selects an item from the list, the selected row will appear with
-  the `checkmark--filled` icon.
+  the `radio-button--checked` icon.
+
+## Default
+
+Default structured lists are used to display information to the user, such as
+pricing plans. These structured list can have internal calls-to-action (CTAs),
+such as a [button](/components/button/usage) or
+a [link](/components/link/usage).
+
+<Row>
+<Column colLg={12}>
+
+![default structured list](images/structured-list-usage-4.png)
+
+<Caption> Default structured list </Caption>
+
+</Column>
+</Row>
+
+## Selectable
+
+Selectable structured lists work well for presenting options to a user in a
+structured manner, such as a set of pricing plans. Selectable tiles may contain
+internal CTAs (like links to docs) if the internal CTA is given its own click
+target. Selectable structured lists have a single-select state working like
+a [radio button](/components/radio-button/usage).
+
+<Row>
+<Column colLg={12}>
+
+![selectable structured list](images/structured-list-usage-5.png)
```

Comment:
I made some comments in your Figma file about some spacing in one of the rows in the structured list in this image.

There are also a few typos: 
- In the first left column there is a typo in the word "Category".
- In the right panel, there is a typo in the word "Washington".
- In the right panel, in the terms, the word should be "agreed" and not agree.

The radio icons are also not aligned correctly.

Reply:
There is still a typo in the word "Washington" in the right panel.

Reply:
Ohh sorry, yes done!